apps/frontend/src/modules/events/components/index.tsx

```diff
@@ -0,0 +1,154 @@
+            },
+        });
+
+        setPropertiesWithDefinitions(
+            propertiesWithDefinitionsResponse.data?.propertiesWithDefinitions as EventPropertyWithDefinition[],
+        );
+        setEvents(eventsResponse.data as EventsQuery);
+    };
+
+    return (
+        <section className="px-[72px] pb-20 pt-20">
+            <Header title="Events" description="The events that you have sent" />
+            <div className="mt-4 h-full w-full space-y-12">
+                <div className="flex justify-between">
+                    <h3 className="font-semibold text-intg-text"></h3>
+
+                    <div className="basis-[30%]">
+                        <TextInput
+                            placeholder="Search by name or email"
+                            value={searchValue}
+                            onChange={(e) => setSearchValue(e.target.value)}
+                            icon={Search}
+                            className="w-full"
+                        />
+                    </div>
+                </div>
+                <div className="text-intg-text">
+                    <Table className="scrollbar-hide table-auto rounded-t-lg">
+                        <TableHead className="bg-intg-bg-9 font-light hover:cursor-pointer">
+                            <TableHeaderCell className="w-1/2">Event Name</TableHeaderCell>
+                            <TableHeaderCell className="text-md flex items-center space-x-1 font-normal">
+                                <span>Count</span>
+                                <span>
+                                    <QuestionIcon />
+                                </span>
+                            </TableHeaderCell>
+                            <TableHeaderCell className="text-md font-normal">Last Seen</TableHeaderCell>
+                        </TableHead>
+                        <TableBody className="border-intg-bg-4">
+                            {filteredEvents?.length ? (
+                                filteredEvents.map((event) => {
+                                    return (
+                                        <TableRow
+                                            key={event.node.id}
+                                            className="cursor-pointer border-l border-r border-intg-bg-4 text-center font-light transition-all duration-300 ease-in"
+                                            onClick={() => {
+                                                setSelectedEvent(event as EventDefinitionCountableEdge);
+                                                handleEventDetails(event as EventDefinitionCountableEdge);
+                                            }}
+                                        >
+                                            <TableCell>{event.node.name}</TableCell>
+                                            <TableCell>
+                                                {format(new Date(event.node.createdAt ?? ""), "MMM dd, yyyy")}
```

Comment:
@afolasope Why are you displaying a date under `Count` column. It should be `volume`